frontend/src/components/home/widgets/LatestUsedAppsWidget.tsx

```diff
@@ -33,21 +33,15 @@ export function LatestUsedAppsWidget() {
           .slice(0, 3)
           .map((app) => (
             <Link key={app.id} to={`/apps/${app.appPubkey}`}>
-              <div className="flex items-center justify-between w-full">
-                <div className="flex items-center justify-center gap-4">
-                  <AppAvatar app={app} className="w-14 h-14 rounded-lg" />
-                  <p className="text-sm font-medium">
-                    {app.name === "getalby.com" ? "Alby Account" : app.name}
-                  </p>
-                </div>
-                <div className="flex items-center justify-center gap-4">
-                  <p className="text-xs text-muted-foreground">
-                    {app.lastEventAt
-                      ? dayjs(app.lastEventAt).fromNow()
-                      : "never"}
-                  </p>
-                  <ChevronRight className="text-muted-foreground w-8 h-8" />
-                </div>
+              <div className="flex items-center justify-between w-full gap-4">
```

Comment:
I don't think the justify-between applies properly now since there are 4 children inside the div. How does it look if you only have shorter app names?

Reply:
It works fine but yeah this `justify-between` is unnecessary, since the app name `p` tag has `flex-1` which will grow and shrink depending on space available

> How does it look if you only have shorter app names?

You can see the "spammer" app in the above after screenshot